draft-quic-atsss-reqs.mkd

```diff
@@ -281,7 +281,9 @@ The traffic that matches an ATSSS rule can be distributed among available access
 - “Load-Balancing”: The traffic associated with the matching flow will be distributed among available access networks following a distribution ratio (e.g., 30% via a first access, 70% via a second access).
 - “Priority-based”: For this mode, accesses are assigned priority levels that indicate which access to be used first. Concretely, the traffic associated with the matching flow will be steered on the access with a high priority till congestion is detected, then the overflow will be forwarded over a low priority access. 
 
-These modes of operations were designed assuming that multipath protocols
+In order to provide the above-mentioned steering modes, the ATSSS service relies upon a dedicated protocol called Performance Measurement Function (PMF) protocol. This protocol is enabled between the UE and the UPF and is meant to provide RTT measurements and reporting access availability/unavailability by the UE to the UPF. PMF is not specific to a given steering method (e.g., MPTCP), but applies to any ATSSS steering method. 
```

Comment:
Rather than introducing the term "steering method", maybe it would be best to explicitly refer to ATSSS v1, then state that the PMF protocol is expected to be or will be reused for v2.

Reply:
v1 and v2 are not introduced yet at this stage of the document. Hence the new wording in 55e7177.